build.gradle

```diff
@@ -115,12 +114,11 @@ dependencies {
 
     implementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
 
-    implementation 'com.nimbusds:nimbus-jose-jwt:8.16' // pinned due to CVE-2019-17195 in 7.8.1 brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
+    implementation 'com.nimbusds:nimbus-jose-jwt'
```

Comment:
sorry I more meant that nimbus will be included authomatically so don't think you need that line at all?

Reply:
removed it